### DISCUSSION TITLE
Release Date @Release Notes (airsdk.harman.com/release_notes)

### DISCUSSION THREAD

**Discussion by JanFW** [airsdk/Adobe-Runtime-Support#666] [Ideas]
Can you please add the date of the release for the **Release Notes** (https://airsdk.harman.com/release_notes)?

**Comment by ajwfrost** [comment]
Done.. thanks for the idea!

**Reply by 2jfw** [reply]
> Done.. thanks for the idea!

Thanks, much appreciated!
